config/routes.js

```diff
@@ -27,4 +28,9 @@ router.get(
   renderer.results
 );
 
+router.get(
+  '/debug',
```

Comment:
Should there be a test for the route given it might become a relied upon feature?